python/src/deltachat/testplugin.py

```diff
@@ -244,10 +247,7 @@ def set_logging_default(self, logging):
         def get_unconfigured_account(self):
             self.offline_count += 1
             tmpdb = tmpdir.join("offlinedb%d" % self.offline_count)
-            ac = self.make_account(tmpdb.strpath, logid="ac{}".format(self.offline_count))
-            ac._evtracker.init_time = self.init_time
```

Comment:
FWIW the init_time setting had no effect whatsoever -- we now sync all accounts's time correctly in make_account()